src/main/java/com/datatheorem/mobileappsecurity/jenkins/plugin/sendbuild/SendBuildAction.java

```diff
@@ -75,9 +59,18 @@
     private final String proxyUsername;
     private final String proxyPassword;
     private final boolean proxyUnsecureConnection;
```

Comment:
those parameters needs to be moved to the initiator in order to use MasterToSlaveFileCallable invoke method